src/main/resources/init.sql

```diff
@@ -37,4 +37,15 @@ CREATE TABLE therapies (
   name varchar(50),
   description varchar(500),
   active boolean
+);
+
+CREATE TABLE therapist (
+id BIGINT AUTO_INCREMENT PRIMARY KEY,
```

Comment:
I recommend to autoformat this block of code, you'll see the columns will be idented and that's important for readability